package.json

```diff
@@ -15,6 +15,7 @@
   },
   "homepage": "https://github.com/cisagov/crossfeed#readme",
   "dependencies": {
+    "@trussworks/react-uswds": "^1.14.0",
```

Comment:
I think we should not install @trussworks/react-uswds in the root package.json, but rather in frontend/package.json. You can do that by running `cd frontend && npm i @trussworks/react-uswds@latest`

Reply:
Yes - makes sense - I see that now. will make the update.